backend/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM node:16 as builder
```

Comment:
For efficient builds this should be 4-step build like I described in the other PR before.

You can use this as an inspiration, but it is a bit off because it was designed to build ARM code on x64-64 platform and such, hence a bit weirder structure than necessary.